docs/pages/components/tile.md

```diff
@@ -147,18 +147,18 @@ Add `role=button` to rendering a tile as a button
 <br>
 
 ## Disabled state
-Add class `is-disabled` and/or `aria-disabled="true"` attribute
+Add class `is-disabled` attribute
 
 {% capture tile %}
-<div class="fd-tile" aria-disabled="true">
+<div class="fd-tile" is-disabled>
```

Comment:
The `is-disabled` needs to be within the quotes for the `class` attribute.

Reply:
I think we should keep the `aria-disabled='true'` in the examples, but just not on line 150 where we say what selector to use.